spec/services/device_name_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe DeviceName do
+  describe '#device_name' do
```

Comment:
Nit: The `#` convention indicates an instance method, but `device_name` is a class member.

```suggestion
  describe '.device_name' do
```

Reply:
@kevinsmaster5 Could you apply the suggestion here?

Reply:
Sorry about that. I thought I had.